projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.ts

```diff
@@ -121,4 +125,12 @@ export class CaseFileViewFieldComponent implements OnInit, AfterViewInit, OnDest
       this.categoriesAndDocumentsSubscription.unsubscribe();
     }
   }
+
+  public isIcpEnabled() {
```

Comment:
@OgunyemiO can you mention the return type of this function, also can you also add some unit test for this.